modules/extract/functions/EXTRACT/fn_EXTRACT_Extract.sqf

```diff
@@ -0,0 +1,97 @@
+#include "script_component.hpp"
+
+if (!isServer) exitWith {};
+
+private _class = _this select 0;
+private _spawnLoc = _this select 1;
+private _alt = _this select 2;
+private _landLoc = _this select 3;
+private _waitFor = _this select 4;
+private _extractLoc = _this select 5;
+private _vehInit = nil;
+
+private _spawnPos = nil;
+private _landPos = nil;
+private _extractPos = nil;
+private _isHelicopter = false;
+
+if (count _this > 6) then {
+  _vehInit = _this select 6;
+};
+
+switch (typeName _spawnLoc) do {
```

Comment:
do not use typeName, use isEqualType instead

https://community.bistudio.com/wiki/isEqualType